src/features/dashboard/components/ApiTokenTable/CopyButton/index.tsx

```diff
@@ -28,12 +29,15 @@ const CopyButton = ({ value, has_admin = false }: TCopyButton) => {
 
   return (
     <React.Fragment>
-      <button
+      <StandaloneCopyRegularIcon
+        fill='#000000'
```

Comment:
@agrim-deriv can we use a variable here instead of static color? So in the future, if we support dark themes we can easily manage.